src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,76 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "movie_id", referencedColumnName = "id")
+    private Movie movie;
+    @ManyToOne
+    @JoinColumn(name = "cinema_hall_id", referencedColumnName = "id")
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+
+    public MovieSession() {
```

Comment:
Don't create a default constructor when it is not needed. If you're not planning to instantiate `MovieSession` without parameters, you should remove this constructor.